src/python/system/environment.py

```diff
@@ -319,8 +324,12 @@ def get_environment_settings_as_string():
       if not environment_variable_value:
         continue
 
-      environment_string += '[Environment] %s = %s\n' % (
-          environment_variable, environment_variable_value)
+      if platform() == 'WINDOWS':
+        environment_string += '[Environment] set "%s=%s"\n' % (
+            quote(environment_variable), quote(environment_variable_value))
+      else:
+        environment_string += '[Environment] export %s="%s"\n' % (
```

Comment:
Do we really need export and set as example here ? We can skip those i think?

Reply:
Leaving them out SGTM.

Reply:
Done